keyboards/rephlex/2k/config.h

```diff
@@ -0,0 +1,74 @@
+/* Copyright 2023 RephlexZero (@RephlexZero)
+SPDX-License-Identifier: GPL-2.0-or-later */
+#pragma once
+
+#ifdef VIA_ENABLE
+#define DYNAMIC_KEYMAP_LAYER_COUNT 1
+#endif
```

Comment:
No need to hide this behind preprocessors, and this could also be moved to info.json
```suggestion
#define DYNAMIC_KEYMAP_LAYER_COUNT 1
```